api-tester/src/main/java/com/revenuecat/apitester/java/StoreProductAPI.java

```diff
@@ -21,8 +21,8 @@ static void check(final StoreProduct product) {
         final String title = product.getTitle();
         final String description = product.getDescription();
         final String subscriptionPeriod = product.getSubscriptionPeriod();
-        List<PurchaseOption> purchaseOptions = product.getPurchaseOptions();
-        PurchaseOption defaultOption = product.getDefaultOption();
+        List<SubscriptionOption> subscriptionOptions = product.getSubscriptionOptions();
+        SubscriptionOption defaultOption = product.getDefaultOption();
```

Comment:
Hmm it's not related to this PR, but I'm going back and forth with renaming this method to `getDefaultSubscriptionOption()`. It's a pretty long name, but I think it makes it clearer, so I'm leaning towards renaming it. Thoughts? (This shouldn't block this PR).